content/docs/user-guide/project-structure/internal-files.md

```diff
@@ -62,6 +62,13 @@ operation.
 
 - `.dvc/tmp/exps`: This directory will contain workspace copies used for
   temporary or parallel <abbr>experiments</abbr> (see `dvc exp run`).
+  
+<a name="sqlite-default">Note 1</a>: To avoid any possible state corruption,by default the SQLite-based 
+remote indexes and state's `links` and `md5s` are cached internally in the 
+repository itself. This may however be overriden if necessary, 
```

Comment:
> the SQLite-based remote indexes

So this note should apply to `.dvc/tmp/index` too?

Reply:
I actually have to double check again but if I'm not mistaken it applies to all three (index, links and md5s) but is controlled by just the 2 config settings.